com.github.Rirusha.Cassette.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id" : "com.github.Rirusha.Cassette",
```

Comment:
it has to be starting with `io.github` as per the doc

Reply:
Changed in last commit